### ISSUE TITLE
Multisite-Enhancements 1.4.0 problem

### ISSUE THREAD

**Issue by gtorborg** [bueltge/wordpress-multisite-enhancements#33]
We have two multisite WP environments at SMU and both have the Multisite Enhancements plugin installed.  However, when doing a routine update from 1.3.8 to 1.4.0 on one of them, the server threw a 500 error.  I uninstalled the plugin which brought the site back up, and attempted download and re-install of 1.4.0, but had the same result.  I meanwhile still had my copy of 1.3.8 on the other site, and simply copied it over to the site where I had to uninstall 1.4.0, and 1.3.8 is now, once again, working properly in both places.  You, however, need to know that 1.4.0 has a bug that causes it to crash.  Looking forward to a new version that doesn't have this bug.  Are there any other such reports?

Gary Torborg

**Comment by bueltge** [comment]
Hi @gtorborg 
Please can you activate the debug mode in WP; `wp-config.php` Set the constant `WP_DEBUG` to `true`. 
After this change please test again and see the error message and get me this as feedback. I think, after this hint you get an idea of the problem and maybe I can help to solve this.

Best Frank.

**Comment by bueltge** [comment]
Hello @gtorborg 
It should now works also under PHP 5.3, please try again.

**Comment by gtorborg** [comment]
Frank:

I turned on the debug mode and upgraded the plugin, successfully and without incident.  I did not crash the server as I did last time I made this attempt.  I have turned debug mode off again and everything has resumed normal operation with the latest version of the plugin.  Thanks for offering to help.


[Description: Description: C:\Documents and Settings\26740328\My Documents\My Pictures\smulogo.bmp]Gary M. Torborg
Application Developer
Southern Methodist University
Dallas, TX
w: 214-768-7211 or 214-768-7957
c:  214-566-1136
gtorborg@smu.edu<mailto:gtorborg@smu.edu>
Context | Learner | Intellection | Analytical | Connectedness
